src/app/containers/LoginPage/index.js

```diff
@@ -48,24 +48,21 @@ function LoginPage() {
       }
       setApiProgress(API_PROGRESS.REQ);
       if (__USE_BACKUP_API__) {
-        const { apiData } = await apiLogin({ username, password });
-        console.log(apiData);
+        const { data: apiData } = await apiLogin({ username, password });
         if (!apiData.data || !apiData.data.token) {
           setPopupState(true);
           setApiProgress(API_PROGRESS.FAILED);
         } else {
-          userInfo.setUserInfo({ ...userInfo.userInfo, userInfo: apiData.data.token });
-          // eslint-disable-next-line react/prop-types
+          setUserInfo({ ...userInfo, token: apiData.data.token });
```

Comment:
`userInfo` has 4 fields, and you set it wrongly here, need to set the `token` not `userInfo`

Reference to the fields inside userInfo `src/shared/context/UserInfo.js`

```js
  const [userInfo, setUserInfo] = React.useState({
    username: null,
    email: null,
    contestPassword: null,
    token: null,
  });
```